features/bootstrap/ContentAlertsContext.php

```diff
@@ -29,6 +29,6 @@ public function iCompleteTheForm()
     public function iShouldSeeAThankYouMessage()
     {
         $this->assertSession()
-            ->elementContains('css', 'h1', 'Thank you for subscribing!');
+            ->elementContains('css', '#thank-you h1', 'Thank you for subscribing!');
```

Comment:
This needs to be reverted. This is part of the code that drives the behat tests.

Reply:
As per the task. this was like this when it was a h2 tag so why can we not do the same for this?